core-bundle/tests/Routing/RouteProviderTest.php

```diff
@@ -664,14 +673,15 @@ private function createPage(string $language, string $alias, bool $fallback = tr
     {
         /** @var PageModel&MockObject $page */
         $page = $this->mockClassWithProperties(PageModel::class);
-        $page->id = random_int(1, 10000);
+        $page->id = ++$this->pageModelAutoIncrement;
         $page->rootId = 1;
         $page->type = 'regular';
         $page->alias = $alias;
         $page->domain = $domain;
         $page->rootLanguage = $language;
         $page->rootIsFallback = $fallback;
         $page->rootUseSSL = 'https' === $scheme;
+        mt_srand($this->pageModelAutoIncrement);
```

Comment:
Why do we need to seed here?

Reply:
So that the `mt_rand()` in the next line is deterministic.

Reply:
Sorry, missed that line completely. :see_no_evil: